src/compiler/scala/tools/nsc/symtab/SymbolLoaders.scala

```diff
@@ -113,9 +121,12 @@ abstract class SymbolLoaders {
   /** Enter class and module with given `name` into scope of `root`
    *  and give them `completer` as type.
    */
-  def enterClassAndModule(root: Symbol, name: String, completer: SymbolLoader) {
-    val clazz = enterClass(root, name, completer)
-    val module = enterModule(root, name, completer)
+  def enterClassAndModule(root: Symbol, name: String, getCompleter: (Symbol, Symbol) => SymbolLoader) {
```

Comment:
`(Symbol, ModuleSymbol) => SymbolLoader` seems within reach to avoid confusion in the future.